controllers/show.go

```diff
@@ -47,13 +49,54 @@ func (sc *ShowController) GetShow(w http.ResponseWriter, r *http.Request) {
 
 	// Needed so that credits are grouped by type
 
+	var scheduledSeasons = make([]myradio.Season, 0)
+	var timeslots = make([]myradio.Timeslot, 0)
+
+	if err != nil {
+		log.Println(err)
+		utils.RenderTemplate(w, sc.config.PageContext, struct{}{}, "404.tmpl")
+		return
+	}
+
+	for _, season := range seasons {
+		_, timeslotsSingleSeason, _ := sm.GetSeason(season.SeasonID)
+		if season.FirstTimeRaw != "Not Scheduled" {
+			scheduledSeasons = append(scheduledSeasons, season)
+			timeslots = append(timeslots, timeslotsSingleSeason...)
+		}
+	}
+	var latestEndTime time.Time
+	var currentTime = time.Now()
+	var latestTimeslot myradio.Timeslot
+	var latestMixcloud bool
+
+	for _, timeslot := range timeslots {
+
+		layout := "02/01/2006 15:04"
+		startTimeRaw, _ := time.Parse(layout, timeslot.StartTimeRaw)
+		log.Println(timeslot.Duration)
+		var endTimeRaw = startTimeRaw.Add(timeslot.Duration)
+		if endTimeRaw.After(latestEndTime) && endTimeRaw.Before(currentTime) {
+			latestEndTime = endTimeRaw
+			latestTimeslot = timeslot
+		}
+	}
+	if strings.HasPrefix(latestTimeslot.MixcloudStatus, "/URY1350/") {
+		latestMixcloud = true
```

Comment:
`latestMixcloud = strings.HasPrefix...` ?

This function appears to be lacking in newlines generally, perhaps a little bit of a spacing rework?